DOCS.md

```diff
@@ -579,49 +579,255 @@ __Arguments__
+| color                | Thread's message color in `RRGGBB` (default blue is `null`) |
+| nicknames            | An array of `{"userid": "1234", "nickname": "John Doe"}`    |
+| muteUntil            | Timestamp until the mute expires or `null`                  |
+| participants         | An array of participants. See below                         |
+| adminIDs             | An array of thread admin IDs                                |
+| folder               | `INBOX`, `ARCHIVED`, `PENDING` or `OTHER`                   |
+| isGroup              | `true` or `false`                                           |
+| montageThread        | `message_thread:000000` or `null` *not yet tested*          |
+| reactionsMuteMode    | `REACTIONS_NOT_MUTED` or `REACTIONS_MUTED`                  |
+| mentionsMuteMode     | `MENTIONS_NOT_MUTED` or `MENTIONS_MUTED`                    |
+| isArchived           | `true` or `false`                                           |
+| isSubscribed         | `true` or `false`                                           |
+| timestamp            | timestamp in miliseconds                                    |
+| snippet              | Snippet's text message                                      |
+| snippetAttachments   | Attachments in snippet                                      |
+| snippetSender        | ID of snippet sender                                        |
+| lastMessageTimestamp | timestamp in milliseconds                                   |
+| lastReadTimestamp    | timestamp in milliseconds or `null`                         |
+| cannotReplyReason    | `null`, `"RECIPIENTS_NOT_LOADABLE"` or `"BLOCKED"`          |
+
+__`participants` format__
+
+`accountType` is one of the following:
+- `"User"`
+- `"Page"`
+- `"UnavailableMessagingActor"`
+- `"ReducedMessagingActor"`
+
+(*there might be more*)
+
+<table>
+<tr>
+<th>Account type</th>
+<th>Key</th>
+<th>Description</th>
+</tr>
+<tr>
+<td rowspan="13"><code>"User"</code></td>
+<td>userID</td>
+<td>ID of user</td>
+</tr>
+<tr>
+<td>name</td>
+<td>Full name of user</td>
+</tr>
+<tr>
+<td>shortName</td>
+<td>Short name of user (most likely first name)</td>
+</tr>
+<tr>
+<td>gender</td>
+<td>Either
+<code>"MALE"</code>,
+<code>"FEMALE"</code>,
+<code>"NEUTER"</code> or
+<code>"UNKNOWN"</code>
+</td>
+</tr>
+<tr>
+<td>url</td>
+<td>URL of the user's Facebook profile</td>
+</tr>
+<tr>
+<td>profilePicture</td>
+<td>URL of the profile picture</td>
+</tr>
+<tr>
+<td>username</td>
+<td>Username of user or
+<code>null</code>
+</td>
+</tr>
+<tr>
+<td>isViewerFriend</td>
+<td>Is the user a friend of you?</td>
+</tr>
+<tr>
+<td>isMessengerUser</td>
+<td>Does the user use Messenger?</td>
+</tr>
+<tr>
+<td>isVerified</td>
+<td>Is the user verified? (Little blue tick mark)</td>
+</tr>
+<tr>
+<td>isMessageBlockedByViewer</td>
+<td>Is the user blocking messages from you?</td>
+</tr>
+<tr>
+<td>isViewerCoworker</td>
+<td>Is the user your coworker?
+<em>(not yet tested)</em>
```

Comment:
For these "not yet tested" ones, can we just leave them out of the docs if we don't know what they do?

Reply:
It's tested, it returns `true` if user is in the same company on workplace (see [here](https://github.com/ravkr/facebook-chat-api/files/1849633/workplace.json.txt))